homeassistant/components/airzone_cloud/climate.py

```diff
@@ -0,0 +1,207 @@
+    """Add Airzone climate from a config_entry."""
+    coordinator: AirzoneUpdateCoordinator = hass.data[DOMAIN][entry.entry_id]
+
+    entities: list[AirzoneClimate] = []
+
+    # Zones
+    for zone_id, zone_data in coordinator.data.get(AZD_ZONES, {}).items():
+        entities.append(
+            AirzoneZoneClimate(
+                coordinator,
+                zone_id,
+                zone_data,
+            )
+        )
+
+    async_add_entities(entities)
+
+
+class AirzoneClimate(AirzoneEntity, ClimateEntity):
+    """Define an Airzone Cloud climate."""
+
+    async def async_turn_on(self) -> None:
+        """Turn the entity on."""
+        params = {
+            API_POWER: {
+                API_VALUE: True,
+            },
+        }
+        await self._async_update_params(params)
+
+    async def async_turn_off(self) -> None:
+        """Turn the entity off."""
+        params = {
+            API_POWER: {
+                API_VALUE: False,
+            },
+        }
+        await self._async_update_params(params)
+
+    async def async_set_temperature(self, **kwargs: Any) -> None:
+        """Set new target temperature."""
+        params: dict[str, Any] = {}
+        if ATTR_TEMPERATURE in kwargs:
+            params[API_SETPOINT] = {
+                API_VALUE: kwargs[ATTR_TEMPERATURE],
+                API_OPTS: {
+                    API_UNITS: TemperatureUnit.CELSIUS.value,
+                },
+            }
+        await self._async_update_params(params)
+
+    @callback
+    def _handle_coordinator_update(self) -> None:
+        """Update attributes when the coordinator updates."""
+        self._async_update_attrs()
+        super()._handle_coordinator_update()
+
+    @callback
+    def _async_update_attrs(self) -> None:
+        """Update climate attributes."""
+        self._attr_current_temperature = self.get_airzone_value(AZD_TEMP)
+        self._attr_current_humidity = self.get_airzone_value(AZD_HUMIDITY)
+        self._attr_hvac_action = HVAC_ACTION_LIB_TO_HASS[
+            self.get_airzone_value(AZD_ACTION)
+        ]
+        if self.get_airzone_value(AZD_POWER):
+            self._attr_hvac_mode = HVAC_MODE_LIB_TO_HASS[
+                self.get_airzone_value(AZD_MODE)
+            ]
+        else:
+            self._attr_hvac_mode = HVACMode.OFF
+        self._attr_max_temp = self.get_airzone_value(AZD_TEMP_SET_MAX)
+        self._attr_min_temp = self.get_airzone_value(AZD_TEMP_SET_MIN)
+        self._attr_target_temperature = self.get_airzone_value(AZD_TEMP_SET)
+
+
+class AirzoneZoneClimate(AirzoneZoneEntity, AirzoneClimate):
+    """Define an Airzone Cloud Zone climate."""
+
+    def __init__(
+        self,
+        coordinator: AirzoneUpdateCoordinator,
+        system_zone_id: str,
+        zone_data: dict,
+    ) -> None:
+        """Initialize Airzone Cloud Zone climate."""
+        super().__init__(coordinator, system_zone_id, zone_data)
+
+        self._attr_name = f"{zone_data[AZD_NAME]}"
```

Comment:
Format string is not needed here:
```suggestion
        self._attr_name = zone_data[AZD_NAME]
```

Reply:
Also -- is this following the new entity naming standards https://developers.home-assistant.io/docs/core/entity#entity-naming? I had a look at the class hierarchy and couldn't see the use of the new has entity name fields. If not, then review that guidance and incorporate.

Reply:
> Also -- is this following the new entity naming standards https://developers.home-assistant.io/docs/core/entity#entity-naming? I had a look at the class hierarchy and couldn't see the use of the new has entity name fields. If not, then review that guidance and incorporate.

You're right, fixed in 57c2107a10001a2276d96b687f381a145806eb70